### PR TITLE
Dockerize application with ddev

### PR DESCRIPTION
**Pull request by nevinsm**

### CODE REVIEW

**Comment by bd-viget** [comment]
@nevinsm When I get to the step to run `ddev artisan migrate`, I get the following error:
```
   Illuminate\Database\QueryException 

  SQLSTATE[08006] [7] could not connect to server: Connection refused
        Is the server running on host "host.docker.internal" (198.19.248.254) and accepting
        TCP/IP connections on port 54320? (Connection: pgsql, SQL: select c.relname as name, n.nspname as schema, pg_total_relation_size(c.oid) as size, obj_description(c.oid, 'pg_class') as comment from pg_class c, pg_namespace n where c.relkind in ('r', 'p') and n.oid = c.relnamespace and n.nspname not in ('pg_catalog', 'information_schema') order by c.relname)

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:813
    809▕                     $this->getName(), $query, $this->prepareBindings($bindings), $e
    810▕                 );
    811▕             }
    812▕ 
  ➜ 813▕             throw new QueryException(
    814▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    815▕             );
    816▕         }
    817▕     }

      +37 vendor frames 

  38  artisan:35
      Illuminate\Foundation\Console\Kernel::handle()

Failed to run artisan migrate: exit status 1
```

Did I miss a step?

**Comment by nevinsm** [comment]
See the local stup docs, it needs the snac server app up and with an imported DB before this one can run.

File: docs/local-setup.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+### Running the server
+
+Once DDEV is installed, run the following command in the root of this project.
+
+```sh
+ddev start
+```
+
+#### IMPORTANT
+This application shares its database with the snac-server application and snac-server must be running and have an imported database before this application will work.
```
Comment (bd-viget) [inline docs/local-setup.md:18]: I think I was looking for something like this at the top of the document, and when I didn't see it, I assumed this was what I needed to do first. Maybe put this at the top, and link to the `snac` repo to set that up first?

I definitely overlooked this though.

**Review by jglass-st — APPROVED** [review]
Looks good! Nice to have laravel dockerized and moving forward.

**Pull request merged**